src/main/java/in/org/projecteka/hiu/consent/ConsentController.java

```diff
@@ -21,5 +22,13 @@
         String requesterId = TokenUtils.decode(authorization);
         return consentService.create(requesterId, consentRequestData);
     }
+
+    @PostMapping("/consent/notification")
+    public Mono<Void> consentNotification(
+            @RequestHeader(value = "Authorization") String authorization,
+            @RequestBody ConsentNotificationRequest consentNotificationRequest) {
+        String requesterId = TokenUtils.decode(authorization);
```

Comment:
why is the `Authorization` header patientId here? Shouldn't it be CM service token?

Reply:
I didn't understand. Consent manager will pass its id(base64 encoded) in the header as authorization. HIU will decode it and check if the patient belongs to the same consent manager.

Reply:
What I meant is, this is service(CM) to service(HIU) call. So Consent manager user id(Patient id) shouldn't come into pitcher here. For now what you can do is something like how we have HIU id and name in HIU properties, similarly we have have CM id or something in CM properties and pass that as Authorization.